src/Apps/Order2/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const BUYER_GUARANTEE_URL =
+  "https://support.artsy.net/s/article/The-Artsy-Guarantee"
```

Comment:
I considered moving the little banner out into its own component - they look identical in details vs checkout - but kept them separate for now.